content/articles/create-a-registration-form-using-react.js-and-connecting-it-with-firebase/index.md

```diff
@@ -16,11 +17,15 @@ npm start
 ```
 
 
-If your setup is complete you will get to see the below screen:-
+After running the commands you will get to see the below screen:-
 
 ![starting](start.png)
```

Comment:
This image link format isn't correct: Here's an example ! [An example of annotation] (/engineering-education/understanding-pascal-voc-dataset/annotation-example.png). Correct this for all instances.

Reply:
done